contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -245,16 +518,49 @@ NXmpes(NXobject):
           varied axis may be for example energy, momentum, spatial coordinate, pump-probe
           delay, spin index, temperature, etc. The axes traces should be linked to the
           actual encoder position in NXinstrument or calibrated axes in NXprocess.
+      energy(NX_NUMBER):
+        exists: recommended
+        doc: |
+          Calibrated energy axis.
+          
+          The calibrated energy axis can be either in binding or kinetic energy.
+          This should be a link to either
+          /entry/process/energy_calibration/calibrated_axis or
+          /entry/process/energy_correction/calibrated_axis.
+        \@type:
+          type: NX_CHAR
+          exists: recommended
+          doc: |
+            The energy can be either stored as kinetic or as binding energy.
+          enumeration: 
+            kinetic: 
+              doc:
+              - |
+                Calibrated kinetic energy axis.
+              - |
+                xref:
+                  spec: ISO 18115-1:2023
+                  term: 3.35
+                  url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:3.35
+            binding: 
+              doc:
+              - |
+                Calibrated binding energy axis.
+              - |
+                xref:
+                  spec: ISO 18115-1:2023
+                  term: 12.16
+                  url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.16
```

Comment:
We had this discussion also in another context: Very often we provide energies as (E-EF), which is a scale not represented by either of the two: It has the same direction as the kinetic energy, but is referenced to the Fermi level. Do you know if this is somehow covered in the ISO standard? Should we explicitly mention this here, or implicitly include it in "kinetic"?

Reply:
I discussed this with SPECS, and there is no real option for this in the ISO standard. The fermi_level_referencing is yet something else, namely whether the spectrometer work function has been taken into account for the kinetic energy, or not. We should not mix these terms.

Reply:
Can you still add this description?

Reply:
Right, for me this comment showed up without this context in the conversations view of the PR. I have added the description now.